autometa/taxonomy/gtdb.py

```diff
@@ -76,17 +78,20 @@ def create_gtdb_db(reps_faa: str, dbdir: str) -> str:
         os.makedirs(dbdir)
 
     logger.debug(f"Merging {len(faa_index):,} faa files.")
-    combined_faa = os.path.join(dbdir, "gtdb.faa")
+    combined_faa = os.path.join(dbdir, "gtdb.faa") 
     with open(combined_faa, "w") as f_out:
         for faa_file, acc in faa_index.items():
-            with open(faa_file) as f_in:
+            with gzip.open(faa_file, "rb") as f_in:
                 for line in f_in:
-                    if line.startswith(">"):
-                        seqheader = line.lstrip(">")
+                    # print(line.decode('utf-8'))
+                    if line.decode('utf-8').startswith(">"):
+                        seqheader = line.decode('utf-8').lstrip(">")
                         line = f"\n>{acc} {seqheader}"
-                    f_out.write(line)
-    logger.debug(f"Combined GTDB faa file written to {combined_faa}")
-    return combined_faa
+                        f_out.write(line)
+                    else:    
+                        f_out.write(line.decode('utf-8'))
```

Comment:
Following suggestions on lines `86+`-`89+`

```suggestion
                    else:    
                        outline = line
                    f_out.write(outline)
```